src/main.cpp

```diff
@@ -1,8 +1,19 @@
 #include <QApplication>
+#include <QStringListModel>
+#include <QStandardItemModel>
+#include "projectviewerwindow.h"
 
 int main(int argc, char *argv[])
 {
     QApplication a(argc, argv);
 
+    QStandardItemModel *model = new QStandardItemModel;
+    model->appendRow(new QStandardItem(QIcon("/home/igor/icon"), "path1"));
+    model->appendRow(new QStandardItem(QIcon("/home/igor/icon"), "path2"));
+    model->appendRow(new QStandardItem(QIcon("/home/igor/icon"), "path3"));
+
+    ProjectViewerWindow *pvw = new ProjectViewerWindow;
```

Comment:
Same here with ProjectViewerWindow. Create an object on a stack, and it will be removed automatically.